addons/mail/tests/test_link_preview.py

```diff
@@ -86,3 +118,35 @@ def test_02_link_preview_create(self):
                     }
                 }]
             )
+
+    def test_get_link_preview_from_url(self):
+        session = requests.Session()
+        url = 'https://thisdomainedoentexist.nothing'
+        image_url = 'https://dummy-image-url.nothing'
+        with patch.object(requests.Session, 'get', _patched_get_html):
```

Comment:
Isn't it possible to loop (with parameter for og_title / image) and use subtests ? this generally leads to shorter tests (and we better see what is varying)